sentry-compose-helper/src/jvmMain/java/io/sentry/compose/viewhierarchy/ComposeViewHierarchyExporter.java

```diff
@@ -0,0 +1,124 @@
+package io.sentry.compose.viewhierarchy;
+
+import androidx.compose.runtime.collection.MutableVector;
+import androidx.compose.ui.geometry.Rect;
+import androidx.compose.ui.layout.ModifierInfo;
+import androidx.compose.ui.node.LayoutNode;
+import androidx.compose.ui.node.Owner;
+import androidx.compose.ui.semantics.SemanticsConfiguration;
+import androidx.compose.ui.semantics.SemanticsModifier;
+import androidx.compose.ui.semantics.SemanticsPropertyKey;
+import io.sentry.ILogger;
+import io.sentry.compose.SentryComposeHelper;
+import io.sentry.internal.viewhierarchy.ViewHierarchyExporter;
+import io.sentry.protocol.ViewHierarchyNode;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@SuppressWarnings("KotlinInternalInJava")
+public final class ComposeViewHierarchyExporter implements ViewHierarchyExporter {
+
+  @NotNull private final ILogger logger;
+
+  public ComposeViewHierarchyExporter(@NotNull final ILogger logger) {
+    this.logger = logger;
+  }
+
+  @Override
+  public boolean export(@NotNull final ViewHierarchyNode parent, @NotNull final Object element) {
+
+    if (!(element instanceof Owner)) {
+      return false;
+    }
+
+    final SentryComposeHelper composeHelper = new SentryComposeHelper(logger);
```

Comment:
`l`: I think this can be a class property, right?

Reply:
but calling it here everytime would mean we'll actually do the reflection bits potentially in a loop (addChildren), right? And also for each error where we capture a view hierarchy this also will be called every time. 

We could also extract it and pass it as a ctor parameter to both ComposeViewHierarchyExporter and ComposeGestureTargetLocator, hence instantiating it only once and for all.

Reply:
No, not really - this is guarded by the if check beforehand. It will only run once per AndroidComposeView. Since we setup the options on app startup, passing it via ctor also means performing reflection on SDK init - and I wouldn't do that. But if you want we could init lazy init it within the `export` call

Reply:
yeah I think lazy init would do. There's still the case that we'll instantiate this every time there's an error and we capture the view hierarchy in compose right?

Reply:
I've added a lazy-init mechanism now, only on the first time when VH is accessed `SentryComposeHelper` is create, following calls will just re-use the same instance then.